Add maintainers team (#1593)
nf-core/nf-co.re: markdown/about.md

- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
- [@Xesus-Abalo](https://github.com/Xesus-Abalo): Xesús M. Abalo

## Maintainers team {#maintainers}

nf-core test data, modules, and pipeline repositories require regular upkeep and maintenance.

> The maintainer's team takes an active role in managing nf-core repositories in collaboration with the wider nf-core community.

Reviewer: ```suggestion
The maintainer's team consist of dedicated and highly active members of the community, that take official additional roles in managing nf-core repositories in collaboration with the wider nf-core community.
```

> The current members of the maintainer's team are:

Reviewer: ```suggestion
The current members of the maintainer's team in 2023 are:
```

- Daniel Lundin (Linnaeus University)
- Edmund Miller (University of Texas at Dallas & Element Biosciences)
- Friederike Hanssen (Quantitative Biology Center, QBiC, University of Tübingen)
- Júlia Mir Pedrol (Quantitative Biology Center, QBiC, University of Tübingen)
- Gisela Gabernet (Quantitative Biology Center, QBiC, University of Tübingen)
- Harshil Patel (Seqera Labs)
- James A. Fellows Yates (Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute & Max Planck Institute - for Evolutionary Anthropology)
- Jasmin Frangenberg (Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute)
- Jon Manning, (Healx Ltd)
- Jose Espinosa-Carrasco (Centre for Genomic Regulation)
- Louisa Perelo (Quantitative Biology Center, QBiC, University of Tübingen)
- Matthias De Smet (Center For Medical Genetics Ghent)
- Matthias Hörtenhuber (Science for Life Laboratory)
- Maxime U. Garcia (Seqera Labs)
- Moritz E. Beber, (Unseen Bio ApS)
- Nicolas Vannieuwkerke (Center For Medical Genetics Ghent)
- Ramprasad Neethiraj (Science for Life Laboratory)
- Rob Syme (Seqera Labs)
- Simon Pearce (Cancer Research UK Manchester Institute)

> - Sofia Stamoul (Karolinska Institute & Karolinska University Hospital)

Reviewer: ```suggestion
- Sofia Stamouli (Karolinska Institute & Karolinska University Hospital)
```

> - Sofia Stamoul (Karolinska Institute & Karolinska University Hospital)

Reviewer: ```suggestion
- Sofia Stamoul (Karolinska Institute & Karolinska University Hospital)

We thank all the institutions and companies of the maintainers team for their support of nf-core.
```

Or somethign like that?

- [@renbot-bio](https://github.com/renbot-bio): Renuka Kudva
- [@Xesus-Abalo](https://github.com/Xesus-Abalo): Xesús M. Abalo

## Maintainers team {#maintainers}

nf-core test data, modules, and pipeline repositories require regular upkeep and maintenance.
The maintainer's team consist of dedicated and highly active members of the community, that take official additional roles in managing nf-core repositories in collaboration with the wider nf-core community.

> The current members of the maintainer's team in 2023 are:

Author: ```suggestion
The current members of the maintainer's team in 2023 are: <!-- TODO: check if year is up to date -->
```

- Daniel Lundin (Linnaeus University)
- Edmund Miller (University of Texas at Dallas & Element Biosciences)
- Friederike Hanssen (Quantitative Biology Center, QBiC, University of Tübingen)
- Júlia Mir Pedrol (Quantitative Biology Center, QBiC, University of Tübingen)
- Gisela Gabernet (Quantitative Biology Center, QBiC, University of Tübingen)
- Harshil Patel (Seqera Labs)
- James A. Fellows Yates (Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute & Max Planck Institute - for Evolutionary Anthropology)
- Jasmin Frangenberg (Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute)
- Jon Manning, (Healx Ltd)
- Jose Espinosa-Carrasco (Centre for Genomic Regulation)
- Louisa Perelo (Quantitative Biology Center, QBiC, University of Tübingen)
- Matthias De Smet (Center For Medical Genetics Ghent)
- Matthias Hörtenhuber (Science for Life Laboratory)
- Maxime U. Garcia (Seqera Labs)
- Moritz E. Beber, (Unseen Bio ApS)
- Nicolas Vannieuwkerke (Center For Medical Genetics Ghent)
- Ramprasad Neethiraj (Science for Life Laboratory)
- Rob Syme (Seqera Labs)
- Simon Pearce (Cancer Research UK Manchester Institute)

> - Sofia Stamouli (Karolinska Institute & Karolinska University Hospital)

Reviewer 2: Why not the link to github?

Reviewer 2: as we have for core team and alumni
